src/core/transactionmanager.js

```diff
@@ -520,6 +521,17 @@ TransactionManager.prototype._resubmit = function(ledgers_, pending_) {
 TransactionManager.prototype._prepareRequest = function(tx) {
   const submitRequest = this._remote.requestSubmit();
 
+  if (tx.hasMultiSigners()) {
+    tx.setSigningPubKey('');
+
+    if (this._remote.local_signing) {
+      tx.tx_json.Signers = tx.getMultiSigners();
+    } else {
+      submitRequest.message.command = 'submit_multisigned';
+      submitRequest.message.Signers = tx.getMultiSigners();
+    }
+  }
+
   if (this._remote.local_signing) {
     tx.sign();
```

Comment:
Sorry, I haven't got all of the context.  Is the question here whether a `sign` RPC command should fail if there's a `Signers` array inside the `tx_json`?  I don't believe I currently check for that, but I certainly could.

If the transaction actually gets submitted then the checking prevents a transaction from being both single- and multi-signed.